index.bs

```diff
@@ -191,7 +192,8 @@ This specification integrates with the [[!HTML]] specification by patching the a
 Navigation response {#navigation-response}
 ----------
 
-At [=process a navigate response=], after step 7 call [$update the Client Hints set$] with the [=relevant settings object=] and |response| as inputs.
```

Comment:
The old link is still working for me (without build errors), but doesn't have the right title. maybe `[=process a navigate response|attempt to populate the history entry's document=]`

Reply:
> The old link is still working for me (without build errors), but doesn't have the right title. maybe `[=process a navigate response|attempt to populate the history entry's document=]`?

It works, but it's no longer DFN exported so the compiler doesn't like it. Is it okay to just keep the old way?

Reply:
Monkeypatch specs indeed run into this problem a lot. Using an `<a href="">` is reasonable, but probably best is to use `<a spec=HTML>populating a session history entry</a>`. The `spec=HTML` says "please ignore any export controls; I know what I'm doing".

You can also use `<pre class="anchors">`, as in this example: https://github.com/WICG/nav-speculation/blob/5d73c9a4a83f642716f109f025a8964ba0a975c4/prefetch.bs#L17

Reply:
<pre class="anchors"> removed the errors, <a spec=HTML> still noted one linker error. Thanks!